app/src/main/java/com/github/se/eventradar/ui/component/ReusableComponents.kt

```diff
@@ -376,6 +393,289 @@ fun CategoryDisplayColumn(
       }
 }
 
+@Composable
+fun ImagePicker(modifier: Modifier, imageUri: Uri?) {
+  Row(
+      modifier = modifier.padding(top = 16.dp),
+      horizontalArrangement = Arrangement.Center,
+      verticalAlignment = Alignment.CenterVertically) {
+        if (imageUri != null) {
+          val imageBitmap = rememberImagePainter(data = imageUri)
```

Comment:
you are using a depracated function which is not the best of practices. I would strong suggest you to use rememberAsyncImagePainter(). Refer to line 101 of EventDetails.kt to see an exmaple of Vick using this up to date function.